### DISCUSSION TITLE
Histoire support

### DISCUSSION THREAD

**Discussion by Akryum** [lost-pixel/lost-pixel#212] [Ideas]
Hey! Lost Pixel looks nice, would be cool to have an integration with [Histoire](https://github.com/histoire-dev/histoire) (disclaimer: I'm the author). What would be needed to make it work? We already have an integration with Percy [here](https://github.com/histoire-dev/histoire/tree/main/packages/histoire-plugin-percy).

**Comment by d-ivashchuk** [comment]
Hey @Akryum! so `Histoire` is a component playground like a storybook or ladle right? I guess the integration should be a no-brainer if you can expose the **meta.json** file with all the info of the stories/components that you have displayed within your app. Although we have chosen not a plugin architecture but native support for the most popular tools, you can see it from our readme and examples. Histoire seems like an interesting addition to our supported portfolio especially both our tools being open source, so it might be a fruitful partnership :D 

here is how we are doing it with Ladle: 

https://github.com/lost-pixel/lost-pixel/blob/8c8e4f6c6d8945532c3dc963f6d4aac571e0d2a9/src/crawler/ladleScreenshots.ts#L39-L55

**Comment by d-ivashchuk** [comment]
@Akryum if you'd have any questions - I'd be glad to support you and make this integration happen!

**Comment by d-ivashchuk** [comment]
and if you will have a need for a call or faster chat - at your convenience here [https://twitter.com/divdev_](https://twitter.com/divdev_)
